extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/internal/lib/structure/devices/deviceParameters/DeviceStateUpdate.java

```diff
@@ -19,32 +21,37 @@
 public interface DeviceStateUpdate {
 
     // Update types
-
+    // TODO: SLAT is in principle the same like OUTPUT, so may it is better to delete SLAT... and change all to OUTPUT..
+    // in certain circumstances it is also better to rename SLAT_ANGLE to e.g. SECONDARY_OUTPUT
     // light
-    public final static String UPDATE_BRIGHTNESS = "brightness";
-    public final static String UPDATE_ON_OFF = "OnOff";
-    public final static String UPDATE_BRIGHTNESS_INCREASE = "brightnessIncrese";
-    public final static String UPDATE_BRIGHTNESS_DECREASE = "brightnessDecrese";
-    public final static String UPDATE_BRIGHTNESS_STOP = "brightnessStop";
-    public final static String UPDATE_BRIGHTNESS_MOVE = "brightnessMove";
+    public final static String OUTPUT = "output";
+    public final static String ON_OFF = "OnOff";
+    public final static String OUTPUT_INCREASE = "outputIncrese";
+    public final static String OUTPUT_DECREASE = "outputDecrese";
+    public final static String OUTPUT_STOP = "outputStop";
+    public final static String OUTPUT_MOVE = "outputMove";
 
     // shades
-    public final static String UPDATE_SLATPOSITION = "slatposition";
-    public static final String UPDATE_SLAT_ANGLE = "slatAngle";
-    public final static String UPDATE_SLAT_INCREASE = "slatIncrese";
-    public final static String UPDATE_SLAT_DECREASE = "slatDecrese";
-    public static final String UPDATE_SLAT_ANGLE_INCREASE = "slatAngleIncrese";
-    public static final String UPDATE_SLAT_ANGLE_DECREASE = "slatAngleDecrese";
-    public final static String UPDATE_OPEN_CLOSE = "openClose";
-    public static final String UPDATE_OPEN_CLOSE_ANGLE = "openCloseAngle";
-    public final static String UPDATE_SLAT_MOVE = "slatMove";
-    public final static String UPDATE_SLAT_STOP = "slatStop";
+    public final static String SLATPOSITION = "slatposition";
+    public static final String SLAT_ANGLE = "slatAngle";
+    public final static String SLAT_INCREASE = "slatIncrese";
+    public final static String SLAT_DECREASE = "slatDecrese";
+    public static final String SLAT_ANGLE_INCREASE = "slatAngleIncrese";
+    public static final String SLAT_ANGLE_DECREASE = "slatAngleDecrese";
+    public final static String OPEN_CLOSE = "openClose";
+    public static final String OPEN_CLOSE_ANGLE = "openCloseAngle";
+    public final static String SLAT_MOVE = "slatMove";
+    public final static String SLAT_STOP = "slatStop";
 
     // sensor data
-    public final static String UPDATE_ACTIVE_POWER = "activePower";
-    public final static String UPDATE_OUTPUT_CURRENT = "outputCurrent";
-    public final static String UPDATE_ELECTRIC_METER = "electricMeter";
     public final static String UPDATE_OUTPUT_VALUE = "outputValue";
+    public final static String UPDATE_DEVICE_SENSOR = "deviceSensor-";
```

Comment:
Is the "-" at the end intended? Because I have seen places in the ode where you explicitly do a string concatenation with "variable - variable - variable"...